src/nvim/buffer_defs.h

```diff
@@ -1426,7 +1426,7 @@ struct statuscol {
   int sign_attr[SIGN_SHOW_MAX + 1];    ///< attributes used for signs
   int truncate;                        ///< truncated width
   bool draw;                           ///< draw statuscolumn or not
-  char fold_text[10];                  ///< text in fold column (%C)
+  char fold_text[9 * 4 + 1];           ///< text in fold column (%C)
```

Comment:
These numbers are a big magic. Why 9, 4 and 1?

Reply:
foldcolumn can be 1-9 (according to docs)
unicode characters take up to 4 bytes
not sure about the +1

Reply:
I would those as defines similar to `SIGN_SHOW_MAX`.